src/reason-textmate/TokenTheme.re

```diff
@@ -115,11 +115,24 @@ let of_yojson = (~defaultBackground, ~defaultForeground, json: Yojson.Safe.t) =>
         | _ => None
         };
 
+      print_endline(json |> Yojson.Safe.to_string);
```

Comment:
Will need to remove debug logging prior to merging (can crash windows when there is no `stdout`/`stderr` attached)

Reply:
Ah thanks for catching that. It's gone now!